### PR TITLE
Add script templates for EditorScenePostImport

### PR DESCRIPTION
**Pull request by jtnicholl**
I got a bit tired of accidentally writing `post_import` instead of `_post_import` or forgetting `@tool`. So, I started copy-pasting from existing scripts. And even then, I accidentally didn't copy the `@tool` part once and wondered why it didn't work.
I figured there should just be a script template. I added a basic template with some comments describing its use and some commented-out example code that sets draw distances, and also a no comments script that omits the comments.

### CODE REVIEW

File: modules/gdscript/editor/script_templates/EditorScenePostImport/basic_import_script.gd [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# meta-description: Basic import script template
+@tool
+extends EditorScenePostImport
+
+
+# Called by the editor when a scene has this script set as the import script in the import tab.
+func _post_import(scene: Node) -> Object:
+	# Modify the contents of the scene upon import. For example, setting up LODs:
+#	(scene.get_node(^"HighPolyMesh") as MeshInstance3D).draw_distance_end = 5.0
+#	(scene.get_node(^"LowPolyMesh") as MeshInstance3D).draw_distance_begin = 5.0
```
Comment (Calinou) [inline modules/gdscript/editor/script_templates/EditorScenePostImport/basic_import_script.gd:10]: Add missing indentation:

```suggestion
	#	(scene.get_node(^"HighPolyMesh") as MeshInstance3D).draw_distance_end = 5.0
	#	(scene.get_node(^"LowPolyMesh") as MeshInstance3D).draw_distance_begin = 5.0
```

Comment (jtnicholl) [reply modules/gdscript/editor/script_templates/EditorScenePostImport/basic_import_script.gd:10]: There is indentation, it's after the #. When commenting out code using the editor shortcut (Ctrl + K) it does it this way, so I formatted it the same way. Should I not?

Comment (Calinou) [reply modules/gdscript/editor/script_templates/EditorScenePostImport/basic_import_script.gd:10]: In that case, I guess it makes sense to keep things as they are.

**Review by vnen — APPROVED** [review]

**Comment by akien-mga** [comment]
Would be good to also have a C# version of the templates. Would you be able to add this @jtnicholl? (CC @godotengine/mono for help)

**Comment by jtnicholl** [comment]
I've added C# versions.

File: modules/mono/editor/script_templates/EditorScenePostImport/basic_import_script.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+using System;
+
+[Tool]
+public partial class _CLASS_ : _BASE_
+{
+    public override Object _PostImport(Node scene)
+    {
+        // Modify the contents of the scene upon import. For example, setting up LODs:
+//        scene.GetNode<MeshInstance3D>("HighPolyMesh").DrawDistanceEnd = 5.0
+//        scene.GetNode<MeshInstance3D>("LowPolyMesh").DrawDistanceBegin = 5.0
```
Comment (akien-mga) [inline modules/mono/editor/script_templates/EditorScenePostImport/basic_import_script.cs:14]: I think these should stay aligned despite the comment characters - IDEs are then typically clever enough to uncomment the line while keeping the text indented.
```suggestion
//      scene.GetNode<MeshInstance3D>("HighPolyMesh").DrawDistanceEnd = 5.0
//      scene.GetNode<MeshInstance3D>("LowPolyMesh").DrawDistanceBegin = 5.0
```

Comment (jtnicholl) [reply modules/mono/editor/script_templates/EditorScenePostImport/basic_import_script.cs]: Got it.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**